server/services/offenceService.ts

```diff
@@ -318,6 +319,32 @@ export default class OffenceService {
     return errors
   }
 
+  setOffenceFineAmount(
+    session: CookieSessionInterfaces.CookieSessionObject,
+    nomsId: string,
+    courtCaseReference: string,
+    offenceFineAmountForm: OffenceFineAmountForm,
+  ) {
+    const errors = validate(
+      offenceFineAmountForm,
+      {
+        fineAmount: 'required|minWholeNumber:0',
```

Comment:
fine amounts can be to two decimal places I think, not just whole numbers

Reply:
it needs to be two decimal places only, a user shouldn't be able to enter 10.3456 for example